devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy

```diff
@@ -155,17 +157,17 @@ private getSupportedButtonValues() {
 private getChildCount() {
 	if (device.getDataValue("model") == "0106-G") {
 		return 6
-	} else if (device.getDataValue("model") == "HY0048" || device.getDataValue("model") == "E-SceneSwitch-EM-3.0") {
+	} else if (device.getDataValue("model") == "HY0048" || device.getDataValue("model") == "E-SceneSwitch-EM-3.0" || device.getDataValue("model") == "HS6SSA-W-EF-3.0") {
 		return 4
-	} else if (device.getDataValue("model") == "cef8701bb8664a67a83033c071ef05f2") {
+	} else if (device.getDataValue("model") == "cef8701bb8664a67a83033c071ef05f2" || device.getDataValue("model") == "HS6SSB-W-EF-3.0") {
 		return 3
 	}
 }
 
 private getCLUSTER_GROUPS() { 0x0004 }
 
 private boolean isHeimanButton() {
-	device.getDataValue("model") == "E-SceneSwitch-EM-3.0"
+	device.getDataValue("model") == "E-SceneSwitch-EM-3.0" || device.getDataValue("model") == "HS6SSA-W-EF-3.0" || device.getDataValue("model") == "HS6SSB-W-EF-3.0"
```

Comment:
It would look better if You defined a variable with model name and then compare it with supported models.
`def modelName = device.getDataValue("model")`